script_library/bootstrap-ansible-if-necessary.sh

```diff
@@ -10,8 +10,11 @@ function install_ansible (){
     local socok8s_workspace=${SOCOK8S_WORKSPACE_BASEDIR}/${SOCOK8S_ENVNAME}-workspace
     if [[ ! -d ${socok8s_workspace}/.ansiblevenv/ ]]; then
         virtualenv ${socok8s_workspace}/.ansiblevenv/
+        source ${socok8s_workspace}/.ansiblevenv/bin/activate
```

Comment:
BTW, could you add explicit `pip install --upgrade pip`?

Reply:
why?

Reply:
Well at least my pip is complaining about it and I have to do it manually

Reply:
can we do that in another commit/PR ?

Reply:
Yeah, in place upgrading pip is more fun.. let's not add it here